src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -190,6 +191,8 @@ func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*con
 		if defaultStartManagementServerFunc != nil {
 			defaultStartManagementServerFunc(s, ctx, ociSpec)
 		}
+		//start network monitor
+		go netmon.StartNetMon(s.ctx, s.sandbox)
```

Comment:
Thanks for your advice. I noticed that within the container struct, there is a "bundle" attribute that corresponds to the root file system path of the container. If a container is created by Moby, the "bundle" path typically includes "/run/containerd/io.containerd.runtime.v2.task/moby...". To determine whether a container is Moby-created or not, I employ a regular expression search to check if the "bundle" contains the keyword "moby". Do you think this will work?